src/foam/u2/view/ScrollTableView.js

```diff
@@ -193,9 +202,30 @@
     function init() {
       this.onDetach(this.data$proxy.listen(this.FnSink.create({ fn: this.updateCount })));
       this.updateCount();
+
+      if ( this.memento )
+        this.currentMemento$ = this.memento.tail$;
+      else
+        this.currentMemento$ = this.memento$;
     },
 
     function initE() {
+      // var self = this;
+
+      if ( this.currentMemento ) {
+        var id = this.currentMemento.head;
+        if ( ! isNaN(id) ) {
+          id = parseInt(this.currentMemento.head);
+        }
```

Comment:
do we use float for ids?

Reply:
float? haven't previously.  are you building ids -   '1' + '.' + '2'. sort of thing?

Reply:
@jlhughes 
no, I don't but if someone uses float then we would need to do `parseFloat` instead of `parseInt`

Reply:
Floats would be very unusual. We support String and Long with GUID and SeqNo decorators.  Let's not worry about floats for now.

Reply:
DAOSummaryView doesn't have an 'id' property, does it?